public/js/src/module/controller-webform.js

```diff
@@ -873,6 +873,12 @@ function _setEventHandlers(survey) {
             );
         });
     }
+    // This actually belongs in gui.js but that module doesn't have access to the form object.
```

Comment:
I've noticed a fair number of cases like this—where certain behavior is either implemented in an unexpected place, or where there's a layer of indirection so it can be in an expected place—specifically because of access to the form object, or lack thereof.

In enketo/enketo-transformer#171 I introduced a (currently dev-only) global `enketo` object. In that PR it's only used by tests and benchmarks run in a browser context, but I think it's worth considering whether a single Enketo-scoped global might be of some benefit across Transformer/Core/Express. There are some things which are **inherently** already global singletons, and I don't think there'd be any shame in making that explicit.

I could see that opening up a lot of opportunities to clean up some very low-hanging tech debt fruit, while also making some existing implicit data dependencies at least a bit more clear.

All of that said, this is of course fine for now.